backend/mailman_client/mailman.py

```diff
@@ -0,0 +1,25 @@
+from dropbox import Dropbox, files
+import string
+
+MAILMAN_CLIENT_ID = '2j1mfqvrkfz4n65'  # not the best way to store this but for testing/simplicity for now
+
+class MailmanClient():
+    def __init__(self, access_token):
+        self.dbx = Dropbox(
+            oauth2_access_token=access_token,
+            app_key=MAILMAN_CLIENT_ID
+        )
+
+    def upload_file_to_dropbox(self, email_contents, path, desired_file_name):
```

Comment:
What type does email_content need to be here?

Reply:
It would be a string. Based on our convo yesterday I believe we said I would be receiving a string of the contents.